visidata/guide.py

```diff
@@ -1,27 +1,36 @@
 '''
 # A Guide to VisiData Guides
-Each guide shows you how to use a particular feature in VisiData.
+Each guide shows you how to use a particular feature in VisiData. Gray guides have not been written yet. Why not contribute one?
 
-  [:keys]Up/Down[/] to move the row cursor
-  [:keys]Enter[/] to view a topic
-  [:keys]Backspace[/] to come back to this list of guides
+- [:keystrokes]Up/Down[/] to move the row cursor
+- [:keystrokes]Enter[/] to view a topic
+- [:keystrokes]Backspace[/] to come back to this list of guides
 '''
 
 import re
 
-from visidata import vd, BaseSheet, Sheet, ItemColumn, Column, VisiData
+from visidata import vd, BaseSheet, Sheet, ItemColumn, Column, VisiData, ENTER, RowColorizer
+from visidata import wraptext
 
+vd.guides = {}  # guidename -> guideobj
+
+@VisiData.api
+def addGuide(vd, guidename, guideobj):
+    vd.guides[guidename] = guideobj
 
 @VisiData.api
 class GuideGuide(Sheet):
     help = __doc__
     columns = [
         ItemColumn('n', 0, type=int),
-        ItemColumn('sheetname', 1, width=0),
+        ItemColumn('guidename', 1, width=0),
```

Comment:
Not sure, but maybe just `name`?  tThe rowtype should be "guides" and the rowdef should be given in a comment too.